### PR TITLE
WIP Preview InSpec docs

### PR DESCRIPTION
**Pull request by IanMadd**
### Description

Preview InSpec docs in docs.chef.io

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/inspec/resources/_index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  [menu.inspec]
+    title = "InSpec Resources Reference"
+    identifier = "inspec/resources/_index.md InSpec Resources Reference"
+    parent = "inspec/resources"
+    weight = 10
++++
+
+The following list of InSpec resources are available.
+
+## OS
```
Comment (kagarmoe) [inline content/inspec/resources/_index.md:15]:
```suggestion
## Linux and Windows Resources
The following resources work on both Linux and Windows operating systems.
```

File: content/inspec/resources/_index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+{{< inspec_resources platform="os" >}}
+
+### Linux
```
Comment (kagarmoe) [inline content/inspec/resources/_index.md:19]:
```suggestion
### Linux
The following resources work on Linux operating systems.
```

File: content/inspec/resources/_index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+{{< inspec_resources platform="linux" >}}
+
+### Windows
```
Comment (kagarmoe) [inline content/inspec/resources/_index.md:23]:
```suggestion
### Windows
The following resources work on Windows operating systems.
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Looks good! Left a suggestion for handling the OS resources explanation

**Comment by IanMadd** [comment]
See #2578

**Pull request closed without merging**